documentation/example-file-usage/example-file-prerequistes.md

```diff
@@ -0,0 +1,89 @@
+# Prerequisites for using the example scripts
+
+In order to run the example scripts, you'll need to install some supporting software onto your machine (don't worry, it's really easy). 
+
+You'll need Python 3, some pip packages, and the AnkerMake slicer. 
+
+If you're on Linux, you'll also need Wine to install the AnkerMake slicer unless you are ok with ripping files from another machine using MacOS or Windows.
```

Comment:
Can we mention

1) that these tools are natively supported on Linux
and
2) Wine is an option for running AnkerMake slicer on Linux.

"ripping files" is really up to the user, and outside the scope.